app/controllers/questions/questions_controller.rb

```diff
@@ -88,6 +88,12 @@ def form_navigation
 
     private
 
+    def unauthenticated_redirect_in_offseason
+      if Rails.configuration.offseason && !current_client
+        redirect_to root_path and return
```

Comment:
`and return` isn't really necessary here, I think -- it's the final expression of a method, so it'll be a `return` anyway.

Reply:
I think I'd like to see some unit test coverage of this method.

Reply:
1) my understanding was that explicit returns after a redirect were required in before_actions so that you're indicating that you shouldn't move forward with the behavior in the next method. redirect_to does not automatically stop the other controller action from being called.